tsung/transactions/new_case.xml

```diff
@@ -0,0 +1,8 @@
+<transaction name="new_case">
+    <request subst="true">
+        <pgsql type="sql">
+            INSERT INTO casedata (domain, owner_id, case_json)
+            VALUES ('%%_domain%%', '%%_user_id%%', '%%generator:case_json%%');
```

Comment:
You also need to link the case and the form by inserting into the `case_form` table. Not sure how easy that will be since you need both the form_id and the case_id.